chapter04/rwd/ex4-4.pl

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/perl
+
+my $last_person;
```

Comment:
the other way of doing this is to use "state" instead of "my" inside the subroutine so the variable persists between invocations